draft-ietf-rtcweb-jsep.xml

```diff
@@ -1822,11 +1822,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               rules used for RTCP feedback. If AVPF is provided, or an
               "a=rtcp-fb" attribute is present, assume AVPF timing,
               i.e., a default value of "trr-int=0". Otherwise, assume
-              that AVPF is being used in an AVP compatible mode and use
-              AVP timing, i.e., "trr-int=4".</t>
+              that AVPF is being used in an AVP compatible mode and
+              use AVPF timing configured with "trr-int=4".</t>
```

Comment:
I think this is right. AVP does not have configurable timing but AVPF does. And it has two main configurations, the one that is AVP compatible which you get by setting trr-int=4 and the one that is the default for AVPF which is not AVP compatible which you get by setting trr-int=0